routes/server.js

```diff
@@ -1,90 +1,99 @@
-const express = require('express');
+const express = require("express");
 const route = express.Router();
-const env = require('dotenv');
-const mongoose = require('mongoose');
+require("dotenv").config();
+const mongoose = require("mongoose");
 
-const home = require('../views/home.hbs');
-const signin = require('../views/signin.hbs');
-const update = require('../views/update.hbs');
-const unsub = require('../views/unsub.hbs');
-const status = require('../views/status.hbs');
+const home = require("../views/home.hbs");
+const signin = require("../views/signin.hbs");
+const update = require("../views/update.hbs");
+const unsub = require("../views/unsub.hbs");
+const status = require("../views/status.hbs");
 
 //Mongoose connections and configs
 //Connecting to mongo db
-mongoose.connect('mongodb://localhost:27017/Newsletter', {useNewUrlParser: true, useUnifiedTopology: true});
+const mongoConnection = process.env.MONGO_CONNECTION_STRING;
+
+mongoose.connect(mongoConnection || "mongodb://localhost:27017/Newsletter", {
```

Comment:
in case mongoConnection is `undefined` your app takes the local mongo connection